lib/bot.js

```diff
@@ -335,6 +347,44 @@ class Bot {
     return null;
   }
 
+  findWebhook(ircChannel) {
+    const discordChannelName = this.invertedMapping[ircChannel.toLowerCase()];
+    if (discordChannelName) {
+      const webhook = this.webhooks[discordChannelName];
+      if (webhook) {
+        return webhook;
+      }
+    }
+    return null;
+  }
+
+  getDiscordAvatar(nick, channel) {
+    const guildMembers = this.findDiscordChannel(channel).guild.members;
+    const findByNicknameOrUsername = caseSensitive =>
+      (member) => {
+        if (caseSensitive) {
```

Comment:
Personally, I would blame Discord because they allow 2 users with the exact same nickname, but that's just my opinion. I wish there was a bot that could block that.